BraveRewardsTests/RewardsNotificationViewBuilderTests.swift

```diff
@@ -0,0 +1,42 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import BraveRewards
+import XCTest
+@testable import BraveRewardsUI
+
+class RewardsNotificationViewBuilderTests: XCTestCase {
+  
+  func testUnhandledNotifications() {
+    // Notifications not handled:
+    XCTAssertNil(RewardsNotificationViewBuilder.get(notification: notification(.adsLaunch)))
+    XCTAssertNil(RewardsNotificationViewBuilder.get(notification: notification(.backupWallet)))
+    XCTAssertNil(RewardsNotificationViewBuilder.get(notification: notification(.failedContribution)))
+    XCTAssertNil(RewardsNotificationViewBuilder.get(notification: notification(.invalid)))
+  }
+  
+  func testActionNotifiication() {
+    XCTAssert(RewardsNotificationViewBuilder.get(notification: notification(.grant)) is WalletActionNotificationView)
+  }
+  
+  func testMessageNotification() {
+    XCTAssert(RewardsNotificationViewBuilder.get(notification: notification(.tipsProcessed)) is WalletActionNotificationView)
```

Comment:
Probably should just move the assertions inside this test into `testActionNotifiication` since there's no "message notification" anymore

Reply:
Oops, thanks for noticing. Updated now